option optimize_for = LITE_RUNTIME;

message TreasureSellRequest
{
 repeated uint32 pos = 1;
}
